### PR TITLE
Turkish Universities (Hacettepe University, Middle East Technical University, Bilkent University) faculty members update

### PR DESCRIPTION
**Pull request by 14327319**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-b.csv [hunk 1]
```diff
@@ -803,10 +803,10 @@ Brygg Ullmer,Louisiana State University,https://www.lsu.edu/eng/cse/people/facul
 Bu-Sung Lee,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=ebslee,TyytTfQAAAAJ
 Bud Mishra,New York University,https://cs.nyu.edu/mishra,kXVBr20AAAAJ
 Budi Arief,University of Kent,https://www.cs.kent.ac.uk/people/staff/ba284,bSZk250AAAAJ
-Bugra Gedik,Bilkent University,http://www.cs.bilkent.edu.tr/~bgedik/homepage/doku.php,ddLcpMQAAAAJ
```
Comment (emeryberger) [inline csrankings-b.csv:806]: Please add this and any similar entries to `old/industry.csv` with the affiliation at the end. Thanks.

File: csrankings-b.csv [hunk 2]
```diff
@@ -803,10 +803,10 @@ Brygg Ullmer,Louisiana State University,https://www.lsu.edu/eng/cse/people/facul
 Bu-Sung Lee,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=ebslee,TyytTfQAAAAJ
 Bud Mishra,New York University,https://cs.nyu.edu/mishra,kXVBr20AAAAJ
 Budi Arief,University of Kent,https://www.cs.kent.ac.uk/people/staff/ba284,bSZk250AAAAJ
-Bugra Gedik,Bilkent University,http://www.cs.bilkent.edu.tr/~bgedik/homepage/doku.php,ddLcpMQAAAAJ
 Bundit Laekhanukit,SUFE,http://cs.mcgill.ca/~blaekh,vmemchMAAAAJ
 Buntarou Shizuki,University of Tsukuba,https://www.iplab.cs.tsukuba.ac.jp/~shizuki/index-e.html,lXG_jWEAAAAJ
-Burcu Can,Hacettepe University,https://web.cs.hacettepe.edu.tr/~burcucan,BalzcSEAAAAJ
+Burak Kurkcu,Hacettepe University,http://web.cs.hacettepe.edu.tr/~bkurkcu,IRcDIOwAAAAJ
+Burkay Genç,Hacettepe University,http://burkay.tk,w-pGWs4AAAAJ
```
Comment (emeryberger) [inline csrankings-b.csv:809]: Appointment not in CS (per CV, "Inst. of Population Studies; Policy and Strategy Studies"). Please remove or clarify why they meet the inclusion criteria.

File: csrankings-e.csv [hunk 3]
```diff
@@ -411,13 +413,14 @@ Eran Omri,Ariel University,http://www.ariel.ac.il/Projects/trp/GeneralInformatio
...
 Eran Yahav,Technion,http://www.cs.technion.ac.il/~yahave,grAfX0MAAAAJ
+Eray Tuzun,Bilkent University,http://www.cs.bilkent.edu.tr/~eraytuzun,D4jovncAAAAJ
+Eray Tüzün,Bilkent University,http://www.cs.bilkent.edu.tr/~eraytuzun,D4jovncAAAAJ
 Erchin Serpedin,Texas A&M at Qatar,https://www.qatar.tamu.edu/programs/ecen/faculty-and-staff/duplicate-of-dr.-mohamed-nounou,THi_ypMAAAAJ
 Erdinç Öztürk,Sabancı University,http://people.sabanciuniv.edu/erdinco,xHYqay0AAAAJ
 Erez Karpas,Technion,https://karpase.net.technion.ac.il,Dya6lhgAAAAJ
 Erez Petrank,Technion,http://www.cs.technion.ac.il/~erez,ME7VLY8AAAAJ
 Erez Zadok,Stony Brook University,http://www3.cs.stonybrook.edu/~ezk,ZG4V_jYAAAAJ
 Ergun Akleman,Texas A&M University,http://www.viz.tamu.edu/faculty/ergun,T5-M6YMAAAAJ
-Erhan Mengusoglu,Hacettepe University,http://cs.hacettepe.edu.tr/files/erhan_mengusoglu_CVa.pdf,CRHfLuoAAAAJ
```
Comment (emeryberger) [inline csrankings-e.csv:420]: Apparently moved to industry, please update `old/industry.csv`.

File: csrankings-f.csv [hunk 4]
```diff
@@ -135,8 +135,6 @@ Fatih Alagöz,Boğaziçi University,https://www.cmpe.boun.edu.tr/~alagoz,5Nku4S8
 Fatima A. Merchant,University of Houston,https://www.uh.edu/technology/departments/et/people/?l=merchant&f=fatima,fUCDUOoAAAAJ
 Fatima Abu Salem,American University of Beirut,http://www.cs.aub.edu.lb/fa21,NOSCHOLARPAGE
 Fatima K. Abu Salem,American University of Beirut,http://www.cs.aub.edu.lb/fa21,NOSCHOLARPAGE
-Fatos T. Yarman-Vural,Middle East Technical University,http://www.ceng.metu.edu.tr/~vural,zPFQ2BwAAAAJ
-Fatos Tünay Yarman Vural,Middle East Technical University,http://www.ceng.metu.edu.tr/~vural,zPFQ2BwAAAAJ
```
Comment (emeryberger) [inline csrankings-f.csv:139]: Please point to something indicating change of affiliation for this person.

File: csrankings-g.csv [hunk 5]
```diff
@@ -546,8 +546,6 @@ Golnaz Badkobeh,Goldsmiths University of London,https://www.gold.ac.uk/computing
 Gonçalo N. Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13269,mRRctaMAAAAJ
 Gonçalo Nuno Gomes Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13269,mRRctaMAAAAJ
 Gonçalo Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13269,mRRctaMAAAAJ
-Gonenc Ercan,Hacettepe University,http://www.gonencercan.com,s7-3H78AAAAJ
-Gönenç Ercan,Hacettepe University,http://www.gonencercan.com,s7-3H78AAAAJ
```
Comment (emeryberger) [inline csrankings-g.csv:550]: Please point to new affiliation if possible.

File: csrankings-v.csv [hunk 6]
```diff
@@ -66,7 +66,6 @@ Vangelis Metsis,Texas State University,http://cs.txstate.edu/~v_m137,7hDSfoQAAAA
 Vania Dimitrova,University of Leeds,https://eps.leeds.ac.uk/computing/staff/184/professor-vania-dimitrova,cDrHc2YAAAAJ
 Vania G. Dimitrova,University of Leeds,https://eps.leeds.ac.uk/computing/staff/184/professor-vania-dimitrova,cDrHc2YAAAAJ
 Varmo Vene,University of Tartu,http://www.ut.ee/en/read-why-varmo-vene-running-dean,AR7YQTUAAAAJ
-Varol Akman,Bilkent University,http://www.cs.bilkent.edu.tr/~akman,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-v.csv:69]: Web page still active; please point to change of affiliation.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-f.csv (hunk 4 above)
Comment (14327319) [reply csrankings-f.csv:139]: She is a former faculty member as indicated here -> https://ceng.metu.edu.tr/former-faculty-members

File: csrankings-v.csv (hunk 6 above)
Comment (14327319) [reply csrankings-v.csv:69]: He is retired -> https://w3.cs.bilkent.edu.tr/en/retired-faculty-members/

File: csrankings-b.csv (hunk 2 above)
Comment (14327319) [reply csrankings-b.csv:809]: New members are officially listed here -> https://www.cs.hacettepe.edu.tr/#faculty_members

File: csrankings-b.csv (hunk 1 above)
Comment (14327319) [reply csrankings-b.csv:806]: updated

File: csrankings-e.csv (hunk 3 above)
Comment (14327319) [reply csrankings-e.csv:420]: updated

File: csrankings-g.csv (hunk 5 above)
Comment (14327319) [reply csrankings-g.csv:550]: updated, moved to industry

**Review by emeryberger — APPROVED** [review]

**Pull request merged**